bench-tx/src/main.rs

```diff
@@ -19,14 +17,15 @@ use miden_objects::{
     Felt,
 };
 use miden_tx::{
-    host::BasicAuthenticator, TransactionExecutor, TransactionHost, TransactionProgress,
+    host::BasicAuthenticator, utils::Serializable, MockDataStore, TransactionExecutor,
+    TransactionHost, TransactionProgress,
```

Comment:
was `Serializable` moved from miden_lib to miden_tx?

Reply:
They are both re-exports from the `miden-objects` export, which comes from `miden-crypto`, so it does not make a difference. They are probably all re-exported in the individual crates in order to have serialization work as a standalone feature on each crate.